packages/web/components/patterns/HighlightBar.tsx

```diff
@@ -62,7 +65,17 @@ export function HighlightBar(props: HighlightBarProps): JSX.Element {
         yAnchorCoordinate={props.anchorCoordinates.pageY}
         preventAutoFocus={false}
       >
-        <BarContent {...props} />
+        <Box>
+          css=
+          {{
+            width: '350px',
+            background: '$grayBg',
+            borderRadius: '4px',
+            border: '1px solid $grayBorder',
+            boxShadow: theme.shadows.cardBoxShadow.toString(),
```

Comment:
here you can just use `boxShadow: '$cardBoxShadow'` since stitches will set these theme variables.

Reply:
okay 👍